model/src/gateway/presence/mod.rs

```diff
@@ -28,9 +28,7 @@ use crate::{
     user::User,
 };
 use serde::{
-    de::{
-        value::MapAccessDeserializer, DeserializeSeed, Deserializer, MapAccess, SeqAccess, Visitor,
-    },
+    de::{DeserializeSeed, Deserializer, SeqAccess, Visitor},
```

Comment:
i think this refactoring is out of scope, but if there's a justification i'm all ears

Reply:
Sorry, I usually do a lot of changes and then commit them all without going sorting out irrelevant changes. Reverted.